core/controllers/admin.py

```diff
@@ -1988,10 +1990,10 @@ def put(self) -> None:
         """Updates the username for a user.
 
         Raises:
-            InvalidInputException. Invalid username.
-            InvalidInputException. The user does not have a profile picture
+            PageNotFoundException. Invalid username.
```

Comment:
One thought, it might be worth changing the name of this to NotFoundException since we're not dealing with just pages any more.

Again, though, this is optional and your call.